src/main/java/com/thebois/views/infoviews/InfoView.java

```diff
@@ -25,7 +25,7 @@
      *
      * @param widgetViews The widget views to render within this view
      */
-    public InfoView(final List<IActorView> widgetViews) {
+    public InfoView(final ArrayList<IActorView> widgetViews) {
```

Comment:
Try using a more generic type like `Iterable` or `Collection`.

Reply:
Just noticed that it actually was a `List` before but you changed it to `ArrayList`. Why?

Reply:
Hmm, I have no good answer to that. Changing it back.